public/controlpanel.php

```diff
@@ -664,6 +664,29 @@ function onUploadImageComplete(data) {
         </form>
         </div>
         */ ?>
+        <div class='component'>
+            <h3>Delete Account</h3>
+            <p>
+                Your account will be deleted after a 14 days grace period.<br>
+                You will be able to cancel your request within that period.<br>
```

Comment:
trying to make it shorter, I suggest replacing these two sentences with just one. Something like this:

> After requesting account deletion you'll have 14 days to cancel your request.